### ISSUE TITLE
GrantOnAll urn is not complete: DuplicateResourceException: Duplicate resource found

### ISSUE THREAD

**Issue by toadies** [Titan-Systems/titan#79]
`Blueprint.plan()` fails due to validation of duplicate resources.

In the example
```
grants = [
    GrantOnAll(priv="SELECT", on_type="TABLE", in_type="SCHEMA", in_name="some_db", to="some_role"),
    GrantOnAll(priv="INSERT", on_type="TABLE", in_type="SCHEMA", in_name="some_db", to="some_role")
]
bp.add(grants)
plan = bp.plan(session)
```

Reviewing the code, I found the 2 grants had identical URNs, thus duplicate error.
```
>>> print(grants[0].urn)
urn:::grant_on_all/SOME_ROLE?on_type=TABLE&in_type=SCHEMA&in_name=some_db
>>> print(grants[1].urn)
urn:::grant_on_all/SOME_ROLE?on_type=TABLE&in_type=SCHEMA&in_name=some_db
```

Using same code as `future_grant_fqn`, I was able to proceed.
```
def grant_on_all_fqn(data: _GrantOnAll):
    in_type = resource_label_for_type(data.in_type)
    in_name = data.in_name
    on_type = resource_label_for_type(data.on_type).upper()
    collection = format_collection_string({"in_name": in_name, "in_type": in_type, "on_type": on_type})
    return FQN(
        name=data.to.name,
        params={
            "priv": data.priv,
            "on": f"{in_type}/{collection}",
        },
    )
```

**Comment by teej** [comment]
I'm able to repro locally.

**Comment by teej** [comment]
Fixed in [v0.8.6](https://github.com/Titan-Systems/titan/releases/tag/v0.8.6). The fix required me to change the URN structure for GrantOnAll resources. The change is not backwards compatible, any plans saved from a previous version won't work.
